src/dev/impl/DevToys/ViewModels/Tools/Text/XmlValidator/XmlValidatorToolViewModel.cs

```diff
@@ -0,0 +1,128 @@
+﻿using System;
+using System.Composition;
+using System.Xml.Schema;
+using DevToys.Api.Core.Settings;
+using DevToys.Api.Tools;
+using DevToys.Core.Threading;
+using DevToys.Models;
+using DevToys.ViewModels.Tools.XmlValidator.Parsing;
+using DevToys.Views.Tools.XmlValidator;
+using Microsoft.Toolkit.Mvvm.ComponentModel;
+using Microsoft.UI.Xaml.Controls;
+
+namespace DevToys.ViewModels.Tools.XmlValidator
+{
+    [Export(typeof(XmlValidatorToolViewModel))]
+    public class XmlValidatorToolViewModel : ObservableRecipient, IToolViewModel
+    {
+        private string? _xsdSchemaData;
+        private string? _xmlData;
+        private XmlParsingResult? _parsedXml;
+        private XsdParsingResult? _parsedXsdScheme;
+        private readonly XmlValidatorStrings _localizedStrings;
```

Comment:
Please move readonly fields above read-write fields, for consistency with the rest of the repository.

Reply:
Done.